stock_picking_batch_extended/__manifest__.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2012-2014 Alexandre Fayolle, Camptocamp SA
+# Copyright 2018-2020 Tecnativa - Carlos Dauden
+# Copyright 2023 FactorLibre - Boris Alias
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Stock batch picking extended",
+    "summary": "Allows manage a lot of pickings in batch",
+    "version": "16.0.1.0.0",
+    "author": "Camptocamp, "
+    "Tecnativa, "
+    "FactorLibre, "
```

Comment:
This migration doesn't qualify your company for being co-author. A significant contribution to the module should be done for that elevated thing. You are already in contributors section, so it's enough.

Reply:
I'm sorry, I just fixed it.